database/src/main/resources/db/migration/V1.72.0__RHCLOUD-21306_ability_to_subscribe_unsubscribe_down_to_the_event_type.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE public.email_subscriptions (
```

Comment:
How and when will the data be migrated from the old table to this one?

Reply:
By simple sql request when we (back + front team) will be ready to use this new mechanismon prod

Reply:
Features enabled with feature flags are usually migrated using a REST endpoint to make sure everything is enabled/migrated at the right time. Let's talk about that later and open a separate PR about it.

Reply:
Ok I created ticket RHCLOUD-24352 to track this point